types/tests/emotion.tsx

```diff
@@ -0,0 +1,13 @@
+import {} from '@emotion/core'
+import styled from '@emotion/styled'
+import React from 'react'
+import tw from '..'
+
+export const Container = styled.div`
+  ${tw`flex flex-col w-full max-w-sm`}
+  & > form {
+    ${tw`w-full`}
+  }
+`
+
+export const cssProp = <div css={tw`bg-red-100`} />
```

Comment:
typescript error on this line

```
Type '{ css: TwStyle; }' is not assignable to type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'.
  Property 'css' does not exist on type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'.
```

Reply:
I'm not getting a type error in my editor here, and `npm run test:types` doesn't show anything. Make sure you're using the workspace version of TS (it's using the 3.9 beta for `// @ts-expect-error`)